src/bp-forums/templates/default/bbpress/form-reply.php

```diff
@@ -108,13 +108,21 @@
 
 								<?php if ( bbp_is_reply_edit() && ( bbp_get_reply_author_id() !== bbp_get_current_user_id() ) ) : ?>
 
-									<label for="bbp_topic_subscription"><?php _e( 'Notify the author of follow-up replies via email', 'buddyboss' ); ?></label>
+									<label for="bbp_topic_subscription"><?php esc_html_e( 'Notify the author of follow-up replies via email', 'buddyboss' ); ?></label>
```

Comment:
update the same condition here for the legacy email and the new one.

Do same for the topic as well